.github/workflows/deploy-site.yaml

```diff
@@ -51,12 +51,8 @@ jobs:
       - name: Setup gcloud
         uses: google-github-actions/setup-gcloud@v1
 
-      - name: Populate list in publicsuffix.org
-        run: |-
-          cp list/public_suffix_list.dat publicsuffix.org/list/
-          cp list/public_suffix_list.dat publicsuffix.org/list/effective_tld_names.dat
-
       - name: Deploy website
         run: |-
-          gsutil -m -q rsync -d -r -x ".git" publicsuffix.org/ gs://${{ secrets.BUCKET_NAME }}/
-          gsutil -m -q setmeta -h "Cache-Control:public, max-age=86400" gs://${{ secrets.BUCKET_NAME }}/**
+          gsutil -m -q -h "Cache-Control:public, max-age=86400" rsync -d -r -x ".git" publicsuffix.org/ gs://${{ secrets.BUCKET_NAME }}/
+          gsutil -q -h "Cache-Control:public, max-age=86400" -h "Content-Type:text/plain; charset=UTF-8" cp list/public_suffix_list.dat gs://${{ secrets.BUCKET_NAME }}/list/public_suffix_list.dat
+          gsutil -q -h "Cache-Control:public, max-age=86400" -h "Content-Type:text/plain; charset=UTF-8" cp list/public_suffix_list.dat gs://${{ secrets.BUCKET_NAME }}/list/effective_tld_names.dat
```

Comment:
This causes an unfortunate race condition: It briefly sets the content type of the list files to `application/octet-stream`, before setting it to its desired value again. Maybe we can exclude the list data files from the rsync to avoid that? If we do so, we probably don't even need to copy the files into the same tree anymore. Excluding them from the rsync should protect them from being deleted by the rsync command.

Reply:
Should not trigger that since I removed the task that copies the files during build in the `publicsuffix.org` folder. Site and list are now sourced from different folders (thus the list is not included in the first `rsync` command).

Reply:
Right, I didn't notice! However, now we have an even bigger problem: The lists will be briefly deleted by the rsync command. We still need to exclude them to protect them from deletion.